src/alfanous-django/templates/aya_search.html

```diff
@@ -14,16 +14,16 @@
 <div class="control-group">
   <div class="controls">
     <label class="checkbox inline">
-    	<input type="checkbox"  name="fuzzy" value="True" {{ params.fuzzy|yesno:"checked," }} /> 
-		{{ _("Fuzzy search") }}
+      <input type="checkbox"  name="fuzzy" value="True" {% if params.query  %}{{ params.fuzzy|yesno:"checked," }}{% else %}checked="yes"{% endif %} />
```

Comment:
what about rename it exact search, and reverse the logic

Reply:
Yeah, 'Exact' is well known for most users.

Reply:
left it as todo, it need logic should be inverted in js. and it requires translation update.